stdlib/public/Concurrency/Clock.swift

```diff
@@ -0,0 +1,67 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Swift
+
+@available(SwiftStdlib 9999, *)
```

Comment:
The `main` branch has [`SwiftStdlib 5.7` defined now](https://github.com/apple/swift/pull/41046), and I think it would make sense to update availability declarations throughout this PR to match that version.